src/terminal/terminal/terminal_priv.h

```diff
@@ -0,0 +1,446 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+#ifndef _GUAC_TERMINAL_PRIV_H
+#define _GUAC_TERMINAL_PRIV_H
```

Comment:
You'll see this in a lot of older code, but underscore prefixes are apparently reserved for internal use by the C compiler/preprocessor when it comes to global identifiers like these. It's not uncommon to do this anyway, but knowing this now, we should avoid it. This should instead by `GUAC_TERMINAL_PRIV_H`.

The underscore prefix is apparently technically only allowed for things like structure members, and then only if it's not followed by a capital letter or a second underscore.